.circleci/config.yml

```diff
@@ -74,28 +74,28 @@ jobs:
       - run:
           name: Run integration tests
           command: |
-            export NEXT_PUBLIC_API_URL=http://localhost:3000/api
-            export SHOW_TOPIC_EXPLORER=true
-            export INH_URL=https://inh-admin-test.hackney.gov.uk
-            export TABLE_NAME=referrals
-            export CONVERSATIONS_TABLE_NAME=conversations
-            export ADDRESSES_API_URL=${ADDRESSES_API_URL}
             export ADDRESSES_API_KEY=${ADDRESSES_API_KEY}
-            export NEXT_PUBLIC_SIGNPOST_ENABLED=true
+            export ADDRESSES_API_URL=${ADDRESSES_API_URL}
             export AIRTABLE_API_KEY=xx
-            export AIRTABLE_PROMPTS_TABLE_NAME="Conversational Prompts"
-            export FSS_PUBLIC_API_URL=http://localhost:8085/fss
             export AIRTABLE_BASE_ID=baseId123
-            export AIRTABLE_TABLE_NAMES="Service directory"
             export AIRTABLE_BASE_URL=http://localhost:8085
             export AIRTABLE_PROMPTS_API_KEY=xx
             export AIRTABLE_PROMPTS_BASE_ID=baseId456
-            export HOTJAR_ID=12345
+            export AIRTABLE_PROMPTS_TABLE_NAME="Conversational Prompts"
```

Comment:
Wonder if we could replace this whole CCI step with something like this:

```
cat .env.sample | xargs -n1 -I{} export {}
```

...because right now we have to edit these env vars both in .env.sample, readme and in here, which probably explains why these files are out of sync. Of course, ones that require real env vars _(like ADDRESSES_API_URL)_ could be exported manually after this command.

Reply:
Possibly.... I'll leave this one to you to experiment with over the next few weeks